imu_sensor_notes.md

```diff
@@ -134,13 +134,15 @@ Based on these we can conclude on the final equation:
 
 Then we use the coefficient to convert the value into degrees°/s (SI: 0.01745 rad/s):
 
-`acc_vector_component = acc_raw_component * acc_coeff`
+`acc_vector_component = (acc_raw_component - uint16_to_int16(cal_gyro_offset)) * acc_coeff`
+
+Here, unlike acceleration, the origin position is indeed the still position and not a scaling value. So it must be also used in the unit calculation, in addition to coeffition.
 
 ### Gyroscope - Rotation (in revolutions/s)
 
 The equation will become:
 
-`acc_vector_component = acc_raw_component * acc_coeff * 0.0027777778`
+`acc_vector_component = (acc_raw_component - uint16_to_int16(cal_gyro_offset)) * acc_coeff * 0.0027777778`
```

Comment:
This looks to me like it's using the gyro offset in calculation of an acceleration vector. Is that correct?

Reply:
Copy paste mistake, I will fix it